pgxn/neon/neon.c

```diff
@@ -35,6 +36,13 @@ _PG_init(void)
 	pg_init_walproposer();
 
 	EmitWarningsOnPlaceholders("neon");
+        InitConsoleConnector();
+}
+
+void
+_PG_fini(void)
+{
+    CleanupConsoleConnector();
```

Comment:
This is dead code, see https://github.com/postgres/postgres/commit/ab02d702ef08343fba30d90fdf7df5950063e8c9

Reply:
Oh interesting, removed.